renderer/components/Bookings.tsx

```diff
@@ -194,66 +206,63 @@ const Bookings = ({ calendarDate }: BookingsProps) => {
   }, [calendarDate]);
 
   return (
-    <div className="relative px-4 py-5 bg-white shadow sm:rounded-lg sm:px-6 dark:bg-dark-container dark:border dark:border-dark-border">
-      <h2 className="text-lg font-medium text-gray-900 dark:text-dark-heading mb-2">
-        Bookings in {currentReadableMonth}
-      </h2>
-      {loading && (
-        <div className="absolute top-5 right-4">
-          <Loader />
-        </div>
-      )}
-      {isTimetrackerLogged ? (
-        <div className="relative overflow-x-auto">
-          <table className="w-full text-sm text-left">
-            <thead className="text-gray-900 dark:text-dark-heading border-b dark:border-gray-700">
-              <tr>
-                <th scope="col" className="pr-6 py-3 font-semibold">
-                  Project
-                </th>
-                <th scope="col" className="px-6 py-3 font-semibold">
-                  Booked
-                </th>
-                <th scope="col" className="px-6 py-3 font-semibold">
-                  Spent
-                </th>
-              </tr>
-            </thead>
-            <tbody>
-              {bookedSpentStatistic.map((project, i) => (
-                <tr key={i} className="border-b dark:border-gray-700">
-                  <td className="pr-6 py-2 text-gray-700 dark:text-dark-main">
-                    {project.project}
+    <>
+      {isTimetrackerLogged && (
+        <div className="relative px-4 py-5 bg-white shadow sm:rounded-lg sm:px-6 dark:bg-dark-container dark:border dark:border-dark-border">
```

Comment:
In my opinion, this condition should be in an index.js like this `{{isTimetrackerLogged && <Bookings />}`.
Because everything that happens above will be executed every time, even if you are not logged in

Reply:
and I've already done hiding widget in another PR

Reply:
https://github.com/ukad-group/timetracker-desktop-client/pull/195